src/deno_json/ts.rs

```diff
@@ -146,6 +148,30 @@ pub fn parse_compiler_options(
   let mut ignored: Vec<String> = Vec::with_capacity(compiler_options.len());
 
   for (key, value) in compiler_options {
+    if key == "runtime" {
```

Comment:
I think 'runtime' shouldn't be under `compilerOptions` as it's not typescript's option, but it's deno specific one.

The config would look like:
```json
{
  "compilerOptions": {
    ...
  },
  "runtime": [
    "deno",
    "browser",
  ]
}
```

Can you move this check to the upper level?

Reply:
Sure.
Done.